x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferencePlugin.java

```diff
@@ -284,12 +285,19 @@ public Collection<?> createComponents(PluginServices services) {
             String elasticInferenceUrl = this.getElasticInferenceServiceUrl(inferenceServiceSettings);
             elasticInferenceServiceComponents.set(new ElasticInferenceServiceComponents(elasticInferenceUrl));
 
+            var authorizationHandler = new ElasticInferenceServiceAuthorizationHandler(
+                elasticInferenceServiceComponents.get().elasticInferenceServiceUrl(),
```

Comment:
```suggestion
                var elasticInferenceServiceComponentsInstance = elasticInferenceServiceComponents.get();
                elasticInferenceServiceComponentsInstance.elasticInferenceServiceUrl(),
```

nit: we could use a variable here and also use it, when instantiating `ElasticInferenceService`